_layouts/home.html

```diff
@@ -33,18 +33,19 @@ <h3 class="usa-card__heading">{{ post.title }}</h3>
         {% endfor %}
     </ul>
 -->
-    <p><a href="{{ site.baseurl }}/news">News</a></p>
+    <p><a href="{{ site.baseurl }}/news">See all News</a></p>
 </div>
 <div class="grid-container">
     <h2>People</h2>
     <p>Meet the humans of CivicActions Accessibility.</p>
     <ul class="usa-card-group">
         {% for author in site.people %}
         <li class="tablet:grid-col-3 usa-card">
+            <a href="{{ site.baseurl }}{{ author.url }}">
             <div class="usa-card__container">
                 <div class="usa-card__media">
                     <div class="usa-card__img">
-                        <a href="{{ site.baseurl }}{{ author.url }}"><img src="{{ site.baseurl }}/assets/img/people/{{ author.image }}" alt="{{ author.name }}"></a>
+                        <img src="{{ site.baseurl }}/assets/img/people/{{ author.image }}" alt="{{ author.name }}">
```

Comment:
@jaube05 looks like axe accessibility testing wants us to give this a better alt text.

Reply:
Yep just saw this error too. Will look into a better alt text for images of people.